### ISSUE TITLE
Selenium tests use deprecated code

### ISSUE THREAD

**Issue by thestoat** [ox-it/ords#786]
I have needed to update the Selenium drives to support more modern versions of Firefox. In so doing I now see that most Selenium calls have been deprecated. Wisdom here: https://seleniumhq.github.io/selenium/docs/api/java/com/thoughtworks/selenium/DefaultSelenium.html states that "The RC interface will be removed in Selenium 3.0. Please migrate to using WebDriver." This will likely need doing at some stage so this issue has been raised to receive a priority assignment for the work.

**Comment by MerielP** [comment]
I'm afraid I don't know enough about the technical side of things to know exactly what the issue is here! (And hence I don't know how urgent it is.)

Howard suggests asking Ewan/Richard Buckner about this - are they likely to be able to shed any light?

**Comment by scottbw** [comment]
Basically, the Selenium code uses an outdated version of Selenium, and needs to be updated and largely rewritten.

**Comment by thestoat** [comment]
Scott speaks the truth. I have put a request in with Sysdev to do the necessary from their end. Ewen has asked me to get Selenium working again - this issue is just to track the work.

**Comment by MerielP** [comment]
OK - thanks. How distinct is this from issue #787? In particular, how does the amount of work it would take to get the current version working again compare to the amount it would take to update and rewrite everything?

**Comment by thestoat** [comment]
Sorry - was a duplicate. The other has been closed.

There are actually a couple of bits here - getting the tests working again and expanding their coverage. Normally, devs love to start anew but in this case I think it is better to fix what is there - Mark did a good job getting it all up and running and we should keep what he did. Moving forward, once the tests have been fixed they should be expanded to include more scenarios.

**Comment by scottbw** [comment]
Yes, the Cuke bits are good (I added some more of those), its just the code that implements them that needs updating to use current Selenium. 

The main problem isn't really the test code itself, but the configuration of the deployment that stopped them from working unattended. (They actually do still work on a local installation)

**Comment by thestoat** [comment]
The tests all run for me now. However, there are two Sysdev issues, 10161860 and 10174642, that will need to be fixed before I will know for sure that tests are working. After that there will need to be work done in replacing deprecated code from the tests
